variants/fcos/okd-rpms/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM release:artifacts as artifacts
+FROM release:machine-config-operator as mcd
```

Comment:
why do we need this layer?

Reply:
mcd layer might be unnecessary, it was introduced just for "backward compatibility" and used for
```
# TODO: check: is it really needed
COPY --from=mcd /usr/bin/machine-config-daemon /binaries/machine-config-daemon
```